packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,127 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <a
+        :class="[
+            $style['o-link'],
+            { [$style['o-link--bold']]: isBold },
+            { [$style['o-link--noDecoration']]: !hasTextDecoration },
+            { [$style['o-link--full']]: isFullWidth }
+        ]"
+        :target="target"
+        :aria-label="ariaLabel"
+        :rel="rel"
+        v-bind="$attrs"
+    >{{ linkText }}</a>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 import tenantConfigs from '../tenants';
 
 export default {
     name: 'VLink',
-    components: {},
+
+    mixins: [VueGlobalisationMixin],
+
     props: {
-        locale: {
+        linkText: {
```

Comment:
Could this not just use the slot content (rather than a prop) when we use the component? Would also be able to pass in icons as part of this slot content then if needed down the line.

```
<my-custom-link>Some example link text</my-custom-link>

Reply:
I think the tricky part is then extracting that text out from the slot in order to also as part of the `aria-label`, if I'm not mistaken, unless you had something in mind for that, @ashleynolan?

Reply:
I think I've found the solution. 
I think I should have been using `aria-description` instead of `aria-label`. This read the text in the slot and then read the description 'opens in new ...'

Reply:
You should be able to get the content of a slot using `this.$slots.default` (if you only have one slot like this), so you could either assign that directly, or sanitise it in a computed prop (to make sure it's just a string of text) and then use the computed in the aria attribute

Reply:
Brilliant! Thanks so much @ashleynolan